### ISSUE TITLE
SSR our 11ty docs

### ISSUE THREAD

**Issue by claviska** [shoelace-style/webawesome#97]
This would be really nice to have, if it's possible.

See: https://lit.dev/blog/2022-02-07-eleventy/

**Comment by claviska** [comment]
I played with this a bit but ran into some errors that are becoming a rabbit hole for me. Here's what I did:

Install the plugin:

```sh
npm i -D @lit-labs/eleventy-plugin-lit
```

Add it to `.eleventy.js`:

```ts
import litPlugin from '@lit-labs/eleventy-plugin-lit';

// ...

export default function (eleventyConfig) {
  // ...

  // SSR
  eleventyConfig.addPlugin(litPlugin, {
    mode: 'worker',
    componentModules: getComponents().map(component => `./dist/${component.path}`)
  });

  // ...
}
```

Then run:

```sh
npm start
```

The first error I saw was this:

```
Error: Unexpected error while rendering lit component in worker thread ReferenceError [Error]: MutationObserver is not defined
```

So I wrapped all mutation observers in `connectedCallback()` with conditionals but then ran into this:

```
Error: Unexpected error while rendering lit component in worker thread ReferenceError [Error]: HTMLElement is not defined
```

Unfortunately, it's referencing the bundled/chunked code so it's hard to follow exactly what's causing it — but the comment above the affected block of code is:

```js
// node_modules/@lit/reactive-element/css-tag.js
```

I'm stopping here because I was hoping this would be more plug and play and I need to focus on other things for next month's release.
